### ISSUE TITLE
It doesn't work with ng-repeat

### ISSUE THREAD

**Issue by bhoomtawath-plinsut** [mike-allison/angular-tablescroll#1]
When I use ng-tablescroll with ng-repeat. The header won't fix and I can't scroll the table.

``` html
<div style="height: 100px; width: 400px">
    <table ng-tablescroll>
      <thead>
        <tr>
          <th>Col 1</th>
          <th>Col 2</th>
          <th>Col 3</th>
          <th>Col 4</th>
          <th>Col 5</th>
          <th>Col 6</th>
        </tr>
      </thead>
      <tbody>
        <tr ng-repeat="a in [1,2,3,4,5,6]">
          <td>asdfasdfasdfafsd</td>
          <td>asdfasdfasdf</td>
          <td>asdfasdfasdfasdgv</td>
          <td>asdfasdfasdfasdgv</td>
          <td>asdfasdfasdfasdgv</td>
          <td>asdfasdfasdfasdgv</td>
        </tr>
      </tbody>
      <tfoot>
        <tr>
          <td colspan="6">Footer Content</td>
        </tr>
      </tfoot>
    </table>
  </div>
```

However; it work again, once I add another static row under the ng-repeat row.

``` html
<tr ng-repeat="a in [1,2,3,4]">
    <td>1</td>
    <td>2</td>
    <td>3</td>
    <td>4</td>
    <td>5</td>
    <td>6</td>
</tr>
<tr>
    <td>asdfasdfasdfafsd</td>
    <td>asdfasdfasdf</td>
    <td>asdfasdfasdfasdgv</td>
    <td>asdfasdfasdfasdgv</td>
    <td>asdfasdfasdfasdgv</td>
    <td>asdfasdfasdfasdgv</td>
</tr>
```

**Comment by suathh** [comment]
Same problem for me it is not working with ng-repeat

**Comment by mikeallisonJS** [comment]
Thanks for the heads up, I'll take a look. Definitely a big issue.

**Comment by mikeallisonJS** [comment]
This should be resolved with the 2.0.1 release
